libnopegl/src/nopegl.h.in

```diff
@@ -249,9 +249,7 @@ struct ngl_node;
 #define NGL_NODE_TEXTUREVIEW            NGLI_FOURCC('T','e','x','V')
 #define NGL_NODE_TIME                   NGLI_FOURCC('T','i','m','e')
 #define NGL_NODE_TIMERANGEFILTER        NGLI_FOURCC('T','R','F','l')
-#define NGL_NODE_TIMERANGEMODECONT      NGLI_FOURCC('T','R','M','C')
-#define NGL_NODE_TIMERANGEMODENOOP      NGLI_FOURCC('T','R','M','N')
-#define NGL_NODE_TIMERANGEMODEONCE      NGLI_FOURCC('T','R','M','1')
+#define NGL_NODE_TIMESEGMENT            NGLI_FOURCC('T','S','e','g')
```

Comment:
I guess this is a leftover from a previous iteration ?

Reply:
Oops yeah, removed